client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx

```diff
@@ -28,7 +28,7 @@ describe('CommentList', () => {
       id: 2,
       author: 'Furter',
       text: 'ho',
-    }),
+    })
```

Comment:
@thewoolleyman in some cases, we tried using docker for linting. I don't like that it as it doesn't seem to lock down the linter versions.